test/Orleans.Serialization.UnitTests/BuiltInCodecTests.cs

```diff
@@ -98,27 +98,13 @@ public class DateTimeTests : FieldCodecTester<DateTime, DateTimeCodec>
         protected override Action<Action<DateTime>> ValueProvider => Gen.DateTime.ToValueProvider();
     }
 
-    public class DateTimeCopierTests : CopierTester<DateTime, DateTimeCopier>
```

Comment:
I think we still need coverage for these types, even if it's not covering a specific codec.

Reply:
I'll address this in a later commit